### ISSUE TITLE
Problem with file downloading

### ISSUE THREAD

**Issue by vnormatov** [aerokube/ggr#240]
Hi Team,
I have two Selenid servers and GGR server, configured to work with them.
I am trying to download a file from Selenoid node.
I use the request like `curl http://test:test-password@ggr.com:4444/download/7d81706d29e4a769afd0166e55cff4d5fa3ac129b1a8476455af8c81fedff570/file.xml
` 
where

- `7d81706d29e4a769afd0166e55cff4d5fa3ac129b1a8476455af8c81fedff570` sessionId
- `ggr.com` GGR server.

And respond looks like :

`<html>`
`<head><title>308 Permanent Redirect</title></head>`
`<body bgcolor="white">`
`<center><h1>308 Permanent Redirect</h1></center>`
`<hr><center>nginx/1.13.12</center>`
`</body>`
`</html>`

why do I get such respond??
File downloading without GGR worked fine.

**Comment by vania-pooh** [comment]
@vnormatov we don't have Nginx inside our images.

**Comment by vnormatov** [comment]
@vania-pooh , thank you for quick answer.
`curl -L` returned `curl: (35) gnutls_handshake() failed: An unexpected TLS packet was received.`
I created selenoid instances on kebernetes and I use ingress to access outside cluster to the services. I think ingress use Nginx and i have to configure it.
Nevertheless, I solved my downloading file problem.
1 using `curl -s http://test:test-password@ggr.com:4444/host/testSessionId` i got `selenoidHost`
2 splited `testSessionId` by 2 substrings, where first substring is md5 sum of selenoid host name and second is md5 sum of `selenoidSessionId`
3 using  `curl http://selenoidHost/download/selenoidSessionId/file.xml` I downloaded file.
